client/consts_p.go

```diff
@@ -34,6 +34,10 @@ const (
 	// of traffic, and sending a ton of packets all at once during a sync
 	// operation is all but guaranteed to get them dropped.
 	UDPSleepSyncTime = time.Second
+
+	// Event log constants.
+	EventLogExpiry   = 48 * time.Hour
+	EventLogMaxCount = 5000
```

Comment:
What's the max amount of memory that can be consumed by these settings? Why 48 hours instead of something longer?

Reply:
I will make it 7 days and 10 MB as suggested below